deploy/snapshot_class.yaml

```diff
@@ -1,8 +1,7 @@
 kind: VolumeSnapshotClass
-apiVersion: snapshot.storage.k8s.io/v1beta1
+apiVersion: snapshot.storage.k8s.io/v1
 metadata:
   name: csi-cstor-snapshotclass
-  namespace: openebs
```

Comment:
Is this a cluster scoped resource?

Reply:
Also, I couldn't find the `VolumeSnapshotClass` in the cstor-operator.yaml in charts

Reply:
yes its a cluster scope

Reply:
yes ,i just checked snapshotclass file its not there, probably i missed to add that file while refactoring some of the changes.
I will add that in this release charts.

Reply:
It should be added to the cstor-operators in charts repo also right?

Reply:
yes